styles/form.css

```diff
@@ -50,3 +50,55 @@ select {
   line-height: 2rem;
   color: var(--button-hover-bg-color);
 }
+
+#uploadBackupForm {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+}
+
+/*TODO: implement custom upload area */
+/* input[type='file'] {
+  display: none;
+} */
```

Comment:
This requires a little more tinkering than I wanted to do in this PR. We have the MVP of upload functionality here, we can mess with its appearance in the style tickets that are pending.